test/unit/helper.spec.js

```diff
@@ -60,6 +60,48 @@ describe('Helpers Unit Tests', () => {
       expect(num, 'to match', /1[,.]0{3}[,.]0{3}[,.]0{2}/);
       expect(num, 'to match', /\${1}/);
     });
+
+    it('should display one cent', () => {
+      const num = helpers.formatFiat(0.01, 'usd');
+      expect(num, 'to match', /0[,.]0{1}1{1}/);
+      expect(num, 'to match', /\${1}/);
+      expect(num, 'not to match', /^< /);
+    });
+
+    it('should round up to one cent', () => {
+      const num = helpers.formatFiat(0.005, 'usd');
+      expect(num, 'to match', /0[,.]0{1}1{1}/);
+      expect(num, 'to match', /\${1}/);
+      expect(num, 'not to match', /^< /);
+    });
+
+    it('should display less than a cent', () => {
+      const num = helpers.formatFiat(0.004, 'usd');
+      expect(num, 'to match', /0[,.]0{1}1{1}/);
+      expect(num, 'to match', /\${1}/);
+      expect(num, 'to match', /^< /);
+    });
+
+    it('should display less than a cent for small numbers', () => {
+      const num = helpers.formatFiat(0.0000001, 'usd');
+      expect(num, 'to match', /0[,.]0{1}1{1}/);
+      expect(num, 'to match', /\${1}/);
+      expect(num, 'to match', /^< /);
+    });
```

Comment:
@valentinewallace see this test case.

Reply:
I meant it shouldn't round up to 1 cent: https://github.com/lightninglabs/lightning-app/pull/1049/files/34a065e95032d60631495c4630f7df5373b3d9f9#diff-ef34ef96c6b5f7c6aaa62b0f4b1ab4fdR71